styles.css

```diff
@@ -30,7 +48,266 @@
 /* ------------------------------- 4. Areas ---------------------------------------- */
 
 /* --------------- 4.1 Header ------------------------- */
+.navbar-outer-container {
+    display: flex;
+    background: rgb(72,69,228);
+    background: linear-gradient(168deg, rgba(72,69,228,1) 0%, rgba(87,34,205,1) 35%);
+}
+
+
+.navbar-items-inner-container {
+    background-color: #5722CD;
+    font-family: "Roboto", sans-serif;
+    text-decoration-color: white;
+    display: flex;
+    padding: 25px;
+    justify-content: end;
+    list-style: none;
+    gap: 100px;
+    color: white;
+
+}
+
+.navbar-items-inner-container a:link {
+    color: white;
+}
+.navbar-items-inner-container a:visited {
+    color: white;
+}
 
 /* --------------- 4.2 Introduction ------------------- */
+.intro-outer-container-container {
+    display: flex;
+    flex-direction: column;
+    background-color: #5722CD;
+    margin-bottom: 25px;
+}
+
+.intro-h1-inner-container {
+    display: flex;
+
+    text-align: center;
+    color: white;
+    margin-top: 100px;
+    margin-right: 400px;
+    margin-left: 400px;
+    margin-bottom: 25px;
+    font-size: 40px;
+}
+
+.intro-inner-container {
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    text-align: center;
+    color: white;
+    font-family: "Roboto", sans-serif;
+    margin-right: 400px;
+    margin-left: 400px;
+    margin-bottom: 25px;
+    font-size: 15px;
+}
+
+.button {
+    display: flex;
+    flex-basis: auto;
+    align-content: center;
+    justify-content: center;
+    background: #ED4385;
+    font-family: "Roboto", sans-serif;
+    color: white;
+    font-size: 10px;
+    text-transform: uppercase;
+    padding: 10px 10px;
+    border-radius: 25px;
+    margin-right: 690px;
```

Comment:
Ai, dit gaat stuk zodra het scherm een andere breedte krijgt 😨 Nooit doen! Als de button in het midden moet staan, kun je het omwikkelende element een flexbox-container maken of het omwikkelende element `text-align: center` geven, dat werkt voor tekst, buttons én afbeeldingen (als ze niet in een div staan)